components/external_server/tests/database/test_reports.py

```diff
@@ -58,7 +57,6 @@ def setUp(self) -> None:
                 {"report_uuid": REPORT_ID2, "subjects": {SUBJECT_ID2: {"metrics": {METRIC_ID3: {}, METRIC_ID4: {}}}}},
             ),
         ]
-        return super().setUp()
```

Comment:
What happened here?

Reply:
Introduced by accident (by the IDE?) in a previous refactor I think, and now removed.